CONTRIBUTING.md

```diff
@@ -1,60 +1,202 @@
 # Contributing
 
-## Code Quality
+1. [Getting Started](#getting-started)
+   - [Discussing Proprietary Features](#discussing-proprietary-features)
+   - [Setup](#setup)
+1. [Developing Components](#developing-components)
+   - [Tools Used](#tools-used)
+   - [Structure](#structure)
+   - [Quality Checklist](#quality-checklist)
+   - [Server-side Rendering Compatibility](#server-side-rendering-compatibility)
+   - [Linting](#linting)
+   - [Icons](#icons)
+   - [Commit Messages](#commit-messages)
+   - [Pull Requests](#pull-requests)
+   - [Testing Strategies](#testing-strategies)
+   - [Unit Tests](#unit-tests)
+   - [Accessibility Testing](#accessibility-testing)
+1. [Documentation](#documentation)
+1. [Breaking Changes](#breaking-changes)
+   - [Components](#components)
+   - [Styles](#styles)
+   - [Deprecating](#deprectating)
+   - [Removal](#removal)
+
+## Getting Started
+
+Cauldron is Deque's design system / pattern library for building accessible ui as part of our products as Deque. Cauldron consists of several parts, [react](./packages/react), [styles](./packages/styles), and [documentation](./docs).
+
+As Cauldron is first and foremost a design system for building products at Deque, this means we won't necessarily have 1:1 parity with other design systems. Patterns that don't exist in Deque product ui won't be added as a component within the design system.
+
+### Discussing proprietary features
+
+This is a public repo, so care needs to be taken to not include details or screenshots for unannounced or unreleased features in Deque products. Any issues or pull requests need to describe work without including specific business logic or feature details directly related to Deque products. If context is necessary to implement a specific Cauldron feature, please open an issue in the private [cauldron-team repo](https://github.com/dequelabs/cauldron-team/issues) and link to it via the issue or pull request.
```

Comment:
```suggestion
This is a public repo, so care needs to be taken to not include details or screenshots for unannounced or unreleased features in Deque products. Any issues or pull requests need to describe work without including specific business logic or feature details directly related to Deque products. If the context is necessary to implement a specific Cauldron feature, please open an issue in the private [cauldron-team repo](https://github.com/dequelabs/cauldron-team/issues) and link to it via the issue or pull request.
```

Reply:
"If _the_ context is necessary..." sounds really strange to me, as that's makes it seems like there's only one context. I'm thinking of context as a more broad concept here since there could be multiple things that might need to be noted. Does that make sense?

Reply:
Makes sense to me!